go/src/koding/klient/app/klient.go

```diff
@@ -461,11 +465,11 @@ func (k *Klient) RegisterMethods() {
 	k.kite.HandleFunc("klient.usage", k.usage.Current)
 
 	// klient os method(s)
-	k.kite.HandleFunc("os.home", kos.Home)
-	k.kite.HandleFunc("os.currentUsername", kos.CurrentUsername)
+	k.handleWithSub("os.home", kos.Home)
```

Comment:
> If yes, why didn't you use proposed solution in your presence logic?

The idea just popped to my head why I was reading this klient.go file, remote.go was pretty raw. If you don't see a benefit then there's no need for changes. :-)